torch/autograd/gradcheck.py

```diff
@@ -299,12 +302,17 @@ def fn(input):
         def fn(input):
             return _as_tuple(func(*input))[i]
 
-        analytical, reentrant, correct_grad_sizes = get_analytical_jacobian(tupled_inputs, o, nondet_tol=nondet_tol)
+        analytical, reentrant, correct_grad_sizes, correct_grad_types = get_analytical_jacobian(tupled_inputs,
+                                                                                                o,
```

Comment:
Is that really a line with just `o,` ? That why automatic linting is weird! haha